src/actions/users/index.ts

```diff
@@ -0,0 +1,25 @@
+import {
+    IUserFetchFulfilledAction,
+    IUserFetchRejectedAction,
+    IInputSearchAction,
+    IUserFilterAction,
+    IUserFetchAction,
+    IUserSortAction,
+    IUser,
+} from '../../interfaces';
+import {
+    USER_FETCH_FULFILLED, USER_FETCH_REJECTED, INPUT_SEARCH, USER_FILTER, USER_FETCH, USER_SORT,
```

Comment:
please line them up as one per line convention followed above.

Reply:
In fact, just use `import * as types from 'type';` and use `types.USER_SOME_ACTION` in the usage places